File: app/src/main/java/com/prvavaja/grocerease/model/Store.kt
package com.prvavaja.grocerease.model
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Store(
    @SerialName("_id") val id: String,
    val brand: String,
    val name: String,
    val imageUrl: String,
    val address: String,
    val phoneNumber: String,
    val coordinates: Coordinates,
    val __v: Int
)

File: app/src/GrocerEase-backend/models/Store.js
const mongoose = require('mongoose');

const storeSchema = new mongoose.Schema({
    brand: { type: String, required: true },
    name: { type: String, required: true },
    imageUrl: { type: String, required: true },
    address: { type: String, required: true },
    phoneNumber: { type: String, required: true },
    coordinates: {
        lat: { type: Number, required: true },
        lng: { type: Number, required: true },
    },
});

module.exports = mongoose.model('Store', storeSchema);
